.github/workflows/pytest.yml

```diff
@@ -60,6 +60,7 @@ jobs:
       - name: Install requirements
         run: |
           pip install -r extra_requirements/requirements-tests.txt
+          pip install git+https://github.com/AxFoundation/strax.git@v1.6.5 --force-reinstall
```

Comment:
After some re-think, since strax is a released package. This should be `pip install strax==v1.6.5`.

You should add `pip install git+https://github.com/XENONnT/base_environment@el7.sr1_cevns_unblind`before installing strax.

Reply:
Done, thanks!

Reply:
Both should be with `--force-reinstall`.

Reply:
Fixed, thanks!